MOBILE_API.xml

```diff
@@ -3160,6 +3165,12 @@
         <param name="heatedMirrorsEnable" type="Boolean" mandatory="false" since="5.0">
             <description>value false means disabled, value true means enabled.</description>
         </param>
+        <param name="climateEnableAvailable" type="Boolean" mandatory="false" since="6.0">
```

Comment:
Per [the proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0213-rc-radio-climate-parameter-update.md), I believe this is supposed to be `climateEnable`.  I think `climateEnableAvailable` is a param within `ClimateControlCapabilities`.  The proposal does not provide a description for the `climateEnable` param.

Reply:
Good catch! Updated @theresalech